ArduPlane/tailsitter.cpp

```diff
@@ -111,16 +111,32 @@ void QuadPlane::tailsitter_output(void)
         }
     }
 
-    // handle VTOL modes
+    // handle Copter controller
     // the MultiCopter rate controller has already been run in an earlier call
     // to motors_output() from quadplane.update(), unless we are in assisted flight
-    if (assisted_flight) {
+    if (assisted_flight && tailsitter_transition_fw_complete()) {
```

Comment:
needs to be tested with transition for AddictionX tailsitter

Reply:
I have done some revision. This is needed because on a mode chance we instantly switch to assisted forward flight, however we need to stay on the copter controller until the first bit of the transition is done, this is what that check adds. It should have gone in with the initial work to add Qassist.